internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -875,7 +881,8 @@ func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 			if err := CheckRegionConfigsPriorityOrder(req.GetReplicationSpecs()); err != nil {
 				return diag.FromErr(err)
 			}
-			if _, _, err := connV2.ClustersApi.UpdateCluster(ctx, projectID, clusterName, req).Execute(); err != nil {
+			// cannot call 2024-10-23 as it can enable ISS autoscaling
```

Comment:
we can say just "latest API" instead of the specific version as in other comments

Reply:
makes sense, adjusting comments